frontend/src/Editor/component-properties-validation.js

```diff
@@ -0,0 +1,70 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
@arpitnath Could you help me figure out why eslint is not able to resolve superstruct? I think it has something to do with the fact that the lib's files inside `node_modules` are in typescript.

Reply:
eslint is not able to resolve the types installed from the lib, specifying the path to the @types dir fixes it.
https://github.com/ToolJet/ToolJet/pull/2782/commits/27946f1d03d643c5b7585c86172468d8b13f94f8